alonzo/impl/src/Cardano/Ledger/Alonzo/TxBody.hs

```diff
@@ -142,11 +142,13 @@ deriving stock instance
   Eq (TxOut era)
 
 instance
-  ( Show (Core.Value era)
+  ( Era era,
+    Show (Core.Value era)
   ) =>
   Show (TxOut era)
   where
-  show = error "Not yet implemented"
+  show (TxOut addr vl dh) =
+    "TxOut (" <> show addr <> " " <> show vl <> " " <> show dh <> ")"
```

Comment:
This is going to clash with master. I don't think i matters what version we choose.

Reply:
I like what is in master better, I'll rebase, thanks!